src/app/containers/MediaPlayer/index.jsx

```diff
@@ -55,15 +55,21 @@ const MediaPlayerContainer = ({ blocks, placeholder }) => {
     isAmp,
     origin,
   });
+  const iframeTitle = translations.mediaAssetPage.mediaPlayer;
```

Comment:
This should probably use `path` from ramda

```suggestion
  const iframeTitle = path(['mediaAssetPage', 'mediaPlayer'], translations);
```

Reply:
Will also need to import `path` into this file